torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -122,13 +123,30 @@ def resize_image_tensor(
     if image.numel() > 0:
         image = image.reshape(-1, num_channels, old_height, old_width)
 
-        image = _FT.resize(
+        align_corners: Optional[bool] = None
+        if interpolation == InterpolationMode.BILINEAR or interpolation == InterpolationMode.BICUBIC:
+            align_corners = False
+        elif antialias:
+            raise ValueError("Antialias option is supported for bilinear and bicubic interpolation modes only")
+
+        dtype = image.dtype
+        need_cast = dtype != torch.float32 and dtype != torch.float64
```

Comment:
This is because the original implementation does:
https://github.com/pytorch/vision/blob/9f024a6e52e1f9a7a588d743f3f383870ce97fbf/torchvision/transforms/functional_tensor.py#L464

Reply:
Nit:

```suggestion
        need_cast = dtype not in (torch.float32, torch.float64)
```